### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by emredogan7**
Added new changeset.  #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,3 +4,5 @@ just a trial.
 Creating a new pull request.
 
 Some detail, bla bla bla.
+
```
Comment (trial-user-mrdogan7) [inline README.md]: unnecessary line.

Comment (trial-user-mrdogan7) [inline README.md:7]: unnecessary line.

File: README.md [hunk 2]
```diff
@@ -4,3 +4,5 @@ just a trial.
...
+Change #2.
```
Comment (trial-user-mrdogan7) [inline README.md:8]: pls give more details.

**Review by trial-user-mrdogan7 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (emredogan7) [reply README.md:8]: I updated bla bla.

Comment (trial-user-mrdogan7) [reply README.md:8]: ok, lgtm.

File: README.md (hunk 1 above)
Comment (trial-user-mrdogan7) [reply README.md:7]: it should be valid.

**Review by trial-user-mrdogan7 — APPROVED** [review]
lgtm.

**Pull request merged**